changelogs/fragments/66837-zabbix-proxy-interface.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - zabbix_proxy - ``interface`` sub-options ``type`` and ``main`` are now deprecated and will be removed in Ansible 2.14.
```

```suggestion
  - zabbix_proxy - ``interface`` sub-options ``type`` and ``main`` are now deprecated and will be removed in Ansible 2.14. Also, the values passed to ``interface`` are now checked for correct types and unexpected keys.
```
After all, if someone passes some other random information to `interfaces`, that will definitely result in errors now.